src/main/java/it/gov/pagopa/payhub/activities/activity/classifications/ClearClassifyIufActivityImpl.java

```diff
@@ -0,0 +1,53 @@
+package it.gov.pagopa.payhub.activities.activity.classifications;
+
+import it.gov.pagopa.payhub.activities.dao.ClassifyDao;
+import it.gov.pagopa.payhub.activities.dto.classifications.ClassifyDTO;
+import it.gov.pagopa.payhub.activities.exception.ClearClassifyIufException;
+import it.gov.pagopa.payhub.activities.utility.Utilities;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Lazy
+@Component
+public class ClearClassifyIufActivityImpl implements ClearClassifyIufActivity {
+    private final ClassifyDao classifyDao;
+
+    public ClearClassifyIufActivityImpl(ClassifyDao classifyDao) {
+        this.classifyDao = classifyDao;
+    }
+
+    /**
+     *
+     * @param classifyDTO dto containing classification to delete
+     * @return boolean true for a successful deletion otherwise false
+     * @throws ClearClassifyIufException specific exception thrown
+     */
+    public boolean deleteClassificationByIuf(ClassifyDTO classifyDTO) throws ClearClassifyIufException {
+        boolean deletedSuccessfully = true;
+        String classification = classifyDTO.getClassificationCode();
+        Long paymentReportingId = classifyDTO.getPaymentReportingId();
+        verifyParameters(paymentReportingId, classification);
```

Comment:
neither of this is necessary, you should just execute a DELETE operation on the DAO giving the input parameters (and the classification TES_NO_MATCH)